### DISCUSSION TITLE
How to use MIDI feature

### DISCUSSION THREAD

**Discussion by odflor** [frntc/Sidekick64#108] [Q&A]
Hi Pals!

I'm so clumsy that I can't make work the MIDI feature of the SD64.

Resuming:

1. Get MIDI soundfonts somewhere (I find this https://ia800401.us.archive.org/view_archive.php?archive=/16/items/500-soundfonts-full-gm-sets/500_Soundfonts_Full_GM_Sets.zip) and put it in a corresponding folder.
2. Configure SD64 to use the soundfont downloaded.
3. Put some MIDI files on a folder.
4. Use browser to select one of these files.
5. Press ENTER to ear music.

What I'm doing wrong?, is not working for me.

Thanks to All and
Best Regards

**Comment by hpingel** [comment]
First you need a C64 software that runs on the 6510 that uses a MIDI cartridge to send off MIDI data. This could be a music demo like "Instinct" by Ate Bit. Before launching that PRG you need to enable MIDI emulation in Sidekick sound menu (F3 I think).

You can't play pure MIDI files (*.mid). This is not the purpose of MIDI emulation on Sidekick. There is no MIDI-File-Player in Sidekick64.

**Reply by odflor** [reply]
Thanks to You, I am understanding the idea of ​​how to use the cartridge.

I think that the FM emulation is on the same way.

The really complicated thing is finding programs that use the emulated cartridges features.

Best Regards and
Thank You very much!
